lib/ffmpex/options/video.ex

```diff
@@ -32,9 +32,12 @@ defmodule FFmpex.Options.Video do
     hwaccel:          %Option{name: "-hwaccel", require_arg: true, contexts: [:input, :per_stream]},
     hwaccel_device:   %Option{name: "-hwaccel_device", require_arg: true, contexts: [:input, :per_stream]},
     hwaccels:         %Option{name: "-hwaccels"},
+    video_size:       %Option{name: "-video_size", require_arg: true, contexts: [:input]},
```

Comment:
For better organization, please put these under a new file `lib/ffmpex/options/devices/libavdevice.ex` and add an `import` in module `FFmpex.Options` macro `__using__/1`.

Remember to link to https://www.ffmpeg.org/ffmpeg-devices.html in the `@moduledoc` of the new module.